CoroutineTemplate/domain/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+def globalConfig = rootProject.extensions.getByName("ext")
+android {
+    compileSdkVersion globalConfig["android_compile_sdk_version"]
+    defaultConfig {
+        minSdkVersion globalConfig["android_min_sdk_version"]
+        targetSdkVersion globalConfig["android_target_sdk_version"]
+        versionCode globalConfig["android_version_code"]
+        versionName globalConfig["android_version_name"]
```

Comment:
Same as above ☝🏼

Reply:
@chornerman I will change in module app as well 👍